server/config.js

```diff
@@ -0,0 +1,27 @@
+const dotenv = require('dotenv');
+dotenv.config();
```

Comment:
It's not best practice to run code in a config file.
Better do it in the main file of your application (app.js in this case)